samples/Sentry.Samples.AspNetCore.Basic/Program.cs

```diff
@@ -32,6 +33,10 @@ public static void Main(string[] args)
                     // An example ASP.NET Core middleware that throws an
                     // exception when serving a request to path: /throw
                     app.UseRouting();
+
+                    // Enable Sentry performance monitoring
+                    app.UseSentryTracing();
```

Comment:
What about `UseSentryTransaction()`? Is it `tracing` that we're doing?
Tracing to me sounds like "distributed tracing" which isn't really added with this. We don't even have it yet and needs to patch outgoing HTTP requests and read incoming HTTP requests (which we can through this hook indeed).

Requested some comments: https://discord.com/channels/621778831602221064/621792783316942878/799409049778913352

Reply:
`UseSentryPerformance()`?

Reply:
I named it 'tracing' because Sentry docs often refer to it as tracing: https://develop.sentry.dev/sdk/unified-api/tracing/

I don't really like `UseSentryPerformance()` and `UseSentryTransaction()` tbh. Maybe `UseSentryPerformanceMonitoring()`, but that's too long.

Reply:
Agree `SentryPerformance` isn't ideal. "Does it make my app faster?". `PerformanceMonitoring` does what it says but, too long also agree.

`SentryTransaction` does what it says and it isnt' too long. The docs you linked I have a PR renaming already. Remember you didn't even find it because you searched for performance?

I guess we keep it SentryTracing for now

Reply:
> SentryTransaction does what it says and it isnt' too long. The docs you linked I have a PR renaming already. Remember you didn't even find it because you searched for performance?

Didn't realize they were renamed. The reason I'm not too keen on `SentryTransaction` is because it doesn't really convey to me what it does. A transaction is a pretty generic concept that can mean different things. I'm not sure that if I were a Sentry user and saw `SentryTransaction` I would think about performance monitoring.

But of course we can rename it if you are confident that's the way to go.